python/railjson_generator/railjson_generator/schema/infra/speed_section.py

```diff
@@ -36,10 +36,14 @@ def add_track_range(self, track, begin, end, applicable_directions):
             )
```

Comment:
Can we have short tests for the new functions? Likely in new test_speed_section.py and test_track_section.py